index.html

```diff
@@ -11,40 +11,42 @@
 
   <body>
     <!-- header -->
-    <header class="container">
-      <div class="row">
-        <div class="col-xs-12">
-            <div class="header-top">
-              <p>
-                <a href="#">Now Hiring:</a> Are you a driven and motivated 1st Line IT Support Engineer?
-              </p>
-            </div>
-            <div class="header-bottom">
-              <img class="logo" src="./images/logo.png" alt="Logo">
-                <nav>
-                  <a href="#" class="active">Home</a>
-                  <a href="#">Company</a>
-                  <a href="#">IT Solutions</a>
-                  <a href="#">Elements</a>
-                  <a href="#">Case Studies</a>
-                  <a href="#">Blog</a>
-                <div class="language" href="#">
-                    <img class="flags" src="./images/flags/en.png" alt="en" title="English">
-                    <span class="main">English</span>
-                  </div>
-                </nav>
-              <div class="search-bar">
-                <form action="#" class="search"> 
-                  <input type="text" placeholder="Search...">
-                  <button type="submit">
-                    <i class="fa fa-search">
-                    </i>
-                  </button>
-                </form>
+    <section style="background-color: #eeeeee;">
+      <header class="container">
+        <div class="row">
+          <div class="col-xs-12">
+              <div class="header-top">
+                <p>
+                  <a class="hover" href="#">Now Hiring:</a> Are you a driven and motivated 1st Line IT Support Engineer?
+                </p>
+              </div>
+              <div class="header-bottom">
+                <img class="logo" src="./images/logo.png" alt="Logo">
+                  <nav>
+                    <a href="#" class="active">Home</a>
+                    <a href="#">Company</a>
+                    <a href="#">IT Solutions</a>
+                    <a href="#">Elements</a>
+                    <a href="#">Case Studies</a>
+                    <a href="#">Blog</a>
+                  <div class="language" href="#">
+                      <img class="flags" src="./images/flags/en.png" alt="en" title="English">
+                      <span class="main">English</span>
+                    </div>
+                  </nav>
+                <div class="search-bar">
+                  <form action="#" class="search"> 
+                    <input type="text" placeholder="Search...">
+                    <button type="submit">
+                      <i class="fa fa-search">
+                      </i>
+                    </button>
```

Comment:
Šitą dalį galima apjungti į vieną `<button type="submit" class="fa fa-search"></button>`

Žiūrint šiek tiek į ateitį, mygtuke reikėtų įrašyti tekstą "Search" ir jam uždėti stilių `font-size: 0px;` (čia dėl accesibility).

Reply:
Rimantai, apjungiau. Tik del Search irasymo neesu tikras kaip tai padaryti, nes HTML irasius i 
<button type="submit" class="fa fa-search">Search</button>

ir pakeitus i font-size: 0px; prapuls ir mano fa-search mygtuko ikona. Dabar yra 20px:

    header .fa, .fas {
        border: none;
        display: inline-block;
        line-height: 20px;
        font-size: 20px;
        background-color: transparent;
    }